onnxruntime/core/session/inference_session.cc

```diff
@@ -1424,32 +1424,30 @@ common::Status InferenceSession::Initialize() {
       // Then the CUDA EP is cached for triggering a ReplayGraph() in Run().
       auto* cuda_ep = execution_providers_.Get(onnxruntime::kCudaExecutionProvider);
       if (cuda_ep && cuda_ep->IsGraphCaptureEnabled()) {
-        if (cuda_ep->IsGraphCaptureEnabled()) {
```

Comment:
Remove this line of 'if' since it is included in the line above. No change of the code inside the block.